docs/main/user-guides/mayastor.md

```diff
@@ -6,6 +6,24 @@ keywords:
 description: Mayastor documentation is hosted and actively maintained at https://mayastor.gitbook.io/introduction/
 ---
 
-:::note
```

Comment:
Could we add in a new line on this page towards the end with think written out? So that it's clear that the mayastor gitbook is a more thorough reference for the mayastor stuff.

Reply:
added.